apps/src/maze/maze.js

```diff
@@ -95,8 +95,22 @@ module.exports = class Maze {
 
     //TODO: Make configurable.
     studioApp().setCheckForEmptyBlocks(true);
+
+    // TODO elijah: move this out into a separate part of the lifecycle
+    if (studioApp()) {
```

Comment:
Line 97 would have already blown up if `studioApp()` was undefined, might as well put that in this if statement as well (or just assume that studioApp will always be available).